### DISCUSSION TITLE
Better than the integrated duplicate utility?

### DISCUSSION THREAD

**Discussion by tobse91** [vale46n1/immich_duplicate_finder#55] [Q&A]
Hello, 

since June immich offers a built in utility to detect duplicates. Is this tool here in any way superior than the built in one?

I've noticed that the built in tool doesnt recognize every duplicate (especially when i have the same picture in my library, but with slightly different quality).

**Comment by marcstae** [comment]
Hi, i also wanted to chip in and ask the same question.
@vale46n1
